content/06.administrative.md

```diff
@@ -30,24 +30,24 @@ AJW is a member of the Scientific Advisory boards for Alexion and DayOne Biophar
 
 **Figure 5. Transcriptomic and immune landscape of pediatric brain tumors**
-A, First two dimensions from UMAP of transcriptome data. Points colored by broad histology. B, Heatmap of with significant GSVA scores for Hallmark gene sets with tumors ordered by cancer group. C, Box plots of quanTIseq estimates of immune cell proportions in select cancer groups with N > 15 tumors. Note: other HGGs and other LGGs have immune cell proportions similar to DMG and pilocytic astrocytoma, respectively, and are not shown. D, Forest plot depicting additive effects of _CD274_ expression, immune cell proportion, and extent of tumor resection on OS of medulloblastoma patients. HRs with 95% confidence intervals and p-values (multivariate Cox) are listed. Significant p-values are denoted with black diamonds. Reference groups are denoted by grey diamonds. Note: the Macrophage M1 HR was 0 (coefficient = -9.90e+4) with infinite upper and lower CIs, and thus was not included in the figure. E, Box plot of _CD274_ expression (log<sub>2</sub> FPKM) for medulloblastomas grouped by subtype. Bonferroni-corrected p-values from Wilcoxon tests are shown. Box plot represents 5% (lower whisker), 25% (lower box), 50% (median), 75% (upper box), and 95% (upper whisker) quantiles. Only stranded RNA-Seq data is plotted.
+A, First two dimensions of transcriptome data UMAP, with points colored by broad histology. B, Heatmap of with significant GSVA scores for Hallmark gene sets with tumors ordered by cancer group. C, Boxplots of quanTIseq estimates of immune cell proportions in cancer groups with N > 15 tumors. Note: other HGGs and other LGGs have immune cell proportions similar to DMG and pilocytic astrocytoma, respectively, and are not shown. D, Forest plot depicting additive effects of _CD274_ expression, immune cell proportion, and extent of tumor resection on OS of medulloblastoma patients. HRs with 95% confidence intervals and p-values (multivariate Cox) are listed. Black diamonds denote significant p-values, and gray diamonds denote reference groups. Note: the Macrophage M1 HR was 0 (coefficient = -9.90e+4) with infinite upper and lower CIs, and thus was not included in the figure. E, Boxplot of _CD274_ expression (log<sub>2</sub> FPKM) for medulloblastomas grouped by subtype. Bonferroni-corrected p-values from Wilcoxon tests are shown. Boxplot represents 5% (lower whisker), 25% (lower box), 50% (median), 75% (upper box), and 95% (upper whisker) quantiles. Only stranded RNA-Seq data is plotted.
```

Comment:
B has something funky in it (emphasis mine)

> B, Heatmap **of with** significant GSVA scores for Hallmark gene sets with tumors ordered by cancer group.

Spot-checking the version of `{msigdbr}` installed on the Docker image, all Hallmark pathways are included in the [GSVA figure data](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/9f8de5b4e07a409363cc71e945f2715c4d789dae/tables/zenodo-upload/figure-5b-data.csv) (yay figure table!). So despite [the filtering we perform](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/9f8de5b4e07a409363cc71e945f2715c4d789dae/figures/scripts/fig5-panels-gsva-umap.R#L179-L194), all pathways are included, and we can say:

> B, Heatmap of GSVA scores for Hallmark gene sets with tumors ordered by cancer group.

(Side note: We _really_ should have done one panel per line in these 😅 )

Reply:
Done in https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/449/commits/f7fddee341dc1bfea2711ddaa457308154beb226